torch/onnx/utils.py

```diff
@@ -306,7 +314,7 @@ def _export(model, args, f, export_params=True, verbose=False, training=False,
             input_names=None, output_names=None, operator_export_type=OperatorExportTypes.ONNX,
             export_type=ExportTypes.PROTOBUF_FILE, example_outputs=None, propagate=False,
             opset_version=None, _retain_param_name=False, do_constant_folding=False,
-            strip_doc_string=True, dynamic_axes=None):
+            strip_doc_string=True, dynamic_axes=None, keep_initializers_as_inputs=None):
```

Comment:
I think we can use `keep_initializers_as_inputs=True` and pass it down (instead of creating `val_keep_init_as_ip`)?

Reply:
I was hoping to keep the code change to a minimum when bringing back the updated default. But sure, I can change the argument default to `keep_initializers_as_inputs=True`.

Reply:
Done.